### PR TITLE
New methods: find (110 B), findIndex (112 B), findLast (109 B), findLastIndex (112 B)

### PR DESCRIPTION
**Pull request by BabkinAleksandr**
[Ramda's find](http://ramdajs.com/docs/#find)
[Ramda's findIndex](http://ramdajs.com/docs/#findIndex)

### CODE REVIEW

**Comment by BabkinAleksandr** [comment]
I'll add `findLast` and `findLastIndex` here, forgot about them

**Comment by BabkinAleksandr** [comment]
Done

File: lib/findLast/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function findLast(fn, list) {
+  var reversed = list.reverse()
+  for (var i = 0; i < reversed.length; i++) {
+    if (fn(reversed[i])) return reversed[i]
+  }
```
[the remark this answers is not in the document]
Comment (BabkinAleksandr) [reply lib/findLast/index.js]: haha, absolutely

**Comment by BabkinAleksandr** [comment]
fixed

**Comment by Beraliv** [comment]
@BabkinAleksandr please check performance tests, you may use `getRandomNumberArrays`

**Comment by BabkinAleksandr** [comment]
find:
aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils_lib__zsh_

findIndex:
aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils_lib__zsh_

findLast:
aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils_lib__zsh_

findLastIndex:
aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils_lib__zsh_

**Comment by Beraliv** [comment]
Do you check the worst cases?

**Comment by BabkinAleksandr** [comment]
Those tests are wrong.

I've made other tests and run it 10 times for each method.
Here are the differences between loops:

### Find

| for       | while           |
| -------|-----------|
| aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ | aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ |

while wins

----------
### FindIndex

| for       | while           |
| -------|-----------|
| aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ | aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ |

while wins

----------

### FindLast

| for       | while           |
| -------|-----------|
| aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ | aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ |

for wins

----------

### FindLastIndex

| for       | while           |
| -------|-----------|
| aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ | aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils__zsh_ |

while wins

**Comment by BabkinAleksandr** [comment]
Conflicts resolved

**Pull request merged**